packages/web-react/src/components/Modal/stories/Modal.tsx

```diff
@@ -0,0 +1,115 @@
+// Because there is no `dist` directory during the CI run
+/* eslint-disable import/no-extraneous-dependencies, import/extensions, import/no-unresolved */
+import React, { useState } from 'react';
+import { ComponentStory } from '@storybook/react';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore: No declaration file
+import icons from '@lmc-eu/spirit-icons/dist/icons';
+import { IconsProvider } from '../../../context';
+import Modal from '../Modal';
+import ModalHeader from '../ModalHeader';
+import ModalBody from '../ModalBody';
+import ModalFooter from '../ModalFooter';
+import { Button } from '../../Button';
+
+const Story: ComponentStory<typeof Modal> = () => {
+  const [defaultOpen, setDefaultOpen] = useState(false);
+  const [longTextOpen, setLongTextOpen] = useState(false);
+
+  return (
+    <IconsProvider value={icons}>
+      <Button onClick={() => setDefaultOpen(true)}>Open Modal</Button>
+      <Button onClick={() => setLongTextOpen(true)}>Open Modal with long content</Button>
+      <Modal id="DemoModalComposed" isOpen={defaultOpen} onClose={() => setDefaultOpen(false)}>
+        <ModalHeader>Modal title</ModalHeader>
```

Comment:
Modal close button must be inside `ModalHeader`, see the [`web` implementation](https://github.com/lmc-eu/spirit-design-system/tree/main/packages/web/src/scss/components/Modal).